pkg/sql/codegen/tensorflow/codegen.go

```diff
@@ -326,7 +326,8 @@ func InitializeAttributes(trainStmt *ir.TrainStmt) error {
 		// Special attributes defined as global variables in `sqlflow_models`
 		modelAttr.Update(attribute.Dictionary{
 			"model.optimizer": {attribute.Unknown, nil, "Specify optimizer", nil},
-			"model.loss":      {attribute.Unknown, nil, "Specify loss", nil}})
+			"model.loss":      {attribute.Unknown, nil, "Specify loss", nil},
+			"model.*":         {attribute.Unknown, nil, "Any model parameters defined in custom models", nil}})
```

Comment:
I added this line so that the attribute checker will recognize newly added parameters in the custom Keras model

Reply:
Do we have a better solution? `model.*` actually disables the attribute checker and defer the diagnostic from compile-time to run-time, which makes it harder for a user to debug her SQL program.

Reply:
I'm afraid there may be no better solutions, people still need to develop a new model and test it with SQLFlow before SQLFlow can generate those attribute checkers.